### PR TITLE
ref Task#14 fix bug

### PR DESCRIPTION
**Pull request by nhahv0902**

### CODE REVIEW

File: app/src/main/java/com/framgia/photoeditor/ui/framgent/adjusment/AdjustFragment.java [hunk 1]
```diff
@@ -98,10 +99,41 @@ public void onStopTrackingTouch(SeekBar seekBar) {
...
+    @OnClick({R.id.image_brightness_cancel, R.id.image_brightness_done})
+    void clickBrightness(View view) {
+        switch (view.getId()) {
+            case R.id.image_brightness_cancel:
+                mLinearControl.setVisibility(View.GONE);
+                mLinearFeature.setVisibility(View.VISIBLE);
+                mImageEdit.setImageBitmap(mBitmap);
+                break;
+            case R.id.image_brightness_done:
+                mLinearControl.setVisibility(View.GONE);
```
Comment (DoanVanToan) [inline app/src/main/java/com/framgia/photoeditor/ui/framgent/adjusment/AdjustFragment.java]: mLinearControl.setVisibility(View.GONE);
 +                mLinearFeature.setVisibility(View.VISIBLE); 
bỏ ra ngoài switch-case

File: app/src/main/res/layout/fragment_adjust.xml [hunk 2]
```diff
@@ -154,4 +155,37 @@
...
+        android:id="@+id/linear_feature_black_white"
+        android:layout_width="match_parent"
+        android:layout_height="@dimen/dp_70"
+        android:background="@color/color_grey_800"
+        android:gravity="center"
+        android:orientation="horizontal"
+        android:paddingLeft="@dimen/dp_16"
+        android:paddingRight="@dimen/dp_16"
+        android:visibility="gone">
+
```
Comment (DoanVanToan) [inline app/src/main/res/layout/fragment_adjust.xml]: remove bl

File: app/src/main/res/layout/fragment_adjust.xml [hunk 3]
```diff
@@ -154,4 +155,37 @@
...
+        <ImageView
+            android:id="@+id/image_undo_black_white"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginLeft="@dimen/dp_16"
+            android:layout_marginRight="@dimen/dp_16"
+            android:padding="@dimen/dp_16"
+            app:srcCompat="@drawable/ic_undo_white"/>
+
```
Comment (DoanVanToan) [inline app/src/main/res/layout/fragment_adjust.xml]: remove bl

File: app/src/main/res/layout/fragment_adjust.xml (hunk 2 above)
Comment (nhahv0902) [inline app/src/main/res/layout/fragment_adjust.xml]: fix

File: app/src/main/res/layout/fragment_adjust.xml (hunk 3 above)
Comment (nhahv0902) [inline app/src/main/res/layout/fragment_adjust.xml]: fix

**Pull request merged**